shotover-proxy/src/message/mod.rs

```diff
@@ -313,27 +319,47 @@ pub enum Value {
     None,
     #[serde(with = "my_bytes")]
     Bytes(Bytes),
+    Ascii(String),
     Strings(String),
-    Integer(i64),
-    Float(f64),
+    Integer(i64, IntSize),
+    Double(f64),
+    Float(f32),
     Boolean(bool),
     Inet(IpAddr),
     List(Vec<Value>),
     Rows(Vec<Vec<Value>>),
     NamedRows(Vec<HashMap<String, Value>>),
     Document(HashMap<String, Value>),
     FragmentedResponse(Vec<Value>),
+    Set(Vec<Value>),
+    Map(Vec<(Value, Value)>),
+    Varint(BigInt),
+    Decimal(BigDecimal),
+    Date(i32),
```

Comment:
Possibly we want something like:
```rust
DateInt(i32)
DateString(String)
```
And equivalent for time.
But what you have is good for now, lets wait and see what our use looks like.